tests/nested/manual/core-early-config/task.yaml

```diff
@@ -0,0 +1,74 @@
+summary: Test that config defaults are applied early when image is created.
+
+# core18 specific test (and nested vm is derived from host system)
+systems: [ubuntu-18.04-64]
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+
+    cleanup_nested_env
+
+    # nested suite installs latest snapd only in the nested system; we need
+    # new snapd on the host as well for prepare-image changes.
+    dpkg -i "$SPREAD_PATH"/../snapd_*.deb
```

Comment:
We are goind this in the spread.yaml file for preparing the test suite. I think it is not needed in the test anymore

Reply:
Indeed, removed.